docs/pricing/index.html

```diff
@@ -35,182 +35,191 @@ <h1 class="c-heading-1">Pricing</h1>
 
 <div class="Pricing-PriceSegments">
 
-  <section id="Pricing-PriceSegments--devTeams">
-    <div class="Pricing-PriceSegments--featureSection title">
-      <span class="title-super">FOR SOFTWARE DEVELOPMENT TEAMS</span>
-      <h2 class="Pricing-PriceSegments--sectionTitle">Dev Environment Automation</h2>
-      <p>Enjoy a simpler development process. Automate your team&apos;s environments. Bring your developers to the cloud and synchronize your environments seamlessly.</p>
-      <span class="footprint prominent">Always free for open source teams.</span>
-    </div>
-    <div class="Pricing-PriceSegments--featureSection features">
-      <h3 class="Pricing-PriceSegments--featuresTitle">Features</h3>
-      <ul>
-        <li class="Pricing-PriceSegments--featureItem">&bull; Automated development environments</li>
-        <li class="Pricing-PriceSegments--featureItem">&bull; Orchestrate your dev infrastructure</li>
-        <li class="Pricing-PriceSegments--featureItem">&bull; Use your own local IDE</li>
-        <li class="Pricing-PriceSegments--featureItem">&bull; Easily share environements</li>
-        <li class="Pricing-PriceSegments--featureItem">&bull; AWS, Vagrant and Azure support </li>
-        <li class="Pricing-PriceSegments--featureItem">&bull; Live code collaboration</li>
-        <li class="Pricing-PriceSegments--featureItem">&bull; Cloud IDE</li>
-        <li class="Pricing-PriceSegments--featureItem">&bull; Analytics (Requires Business Add-On)</li>
-      </ul>
-    </div>
-    <aside class="Pricing-PriceSegments--priceSection">
-      <form>
-        <div class="Dropdown">
-          <a href="#" class="Dropdown-selection">51-100 Monthly Active Developers</a>
-          <div class="Dropdown-options">
-            <a href="#" attr-value="1-10">1-10 Monthly Active Developers</a>
-            <a href="#" attr-value="11-50">11-50 Monthly Active Developers</a>
-            <a href="#" attr-value="51-100">51-100 Monthly Active Developers</a>
-            <a href="#" class="disabled">for 100+ Please contact us...</a>
+  <div itemscope itemtype="http://schema.org/Product">
```

Comment:
I was thinking about it, but decided that the pricing page captures 4 different "products" (i.e. Dev Environment Automation)